js/ConfidenceSliderModel.js

```diff
@@ -129,6 +146,13 @@ export default class ConfidenceSliderModel extends SliderModel {
       _scaleStart: this.linkedModel.get('_scaleStart'),
       _scaleEnd: this.linkedModel.get('_scaleEnd')
     });
+    this._listenToLinkedModel();
+    if (!this.linkedModel.get('_isSubmitted')) {
+      this.set('_isEnabled', false);
+      this.set('body', this.get('disabledBody'));
+    } else {
+      this.set('_linkedModelSelectedIndex', this.linkedModel.get('_selectedItem').index);
+    }
```

Comment:
Rather than duplicating the code, perhaps call as above:
```suggestion
    this._listenToLinkedModel();
    this.updateFromLinkedModel();
```